README.md

```diff
@@ -171,9 +171,10 @@ We welcome and ask for your contribution. If you would like to contribute to
 Mender, please read our guide on how to best get started [contributing code or
 documentation](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md).
 
-### Community maintainers
-
-A list of community maintainers and the description of the role is located [here](https://github.com/mendersoftware/meta-mender-community/wiki/Community-maintainers).
+Community contributed layers will be co-maintained by the Mender team on a best-effort
+base. Coordination of reviews and merges happens case by case and based on responsiveness,
+respectively activity. Reviews and sign-offs by the original contributors are explicitly
+not required for further contributions, in order to not block development and maintenance.
```

Comment:
Hmm, are you sure about this? Thomas was adamant when we first started Mender that *all* contributions must be signed off.

Reply:
Ah, wait, maybe I misunderstood you: You meant: When a *new* contributor makes a PR, the old contributor doesn't have to sign *that* code off?

Reply:
Yep. Bad wording, I guess. Will replace with "acknowledge". The git sign-offs are definitely still required.